requirements/pip.txt

```diff
@@ -72,12 +72,12 @@ git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.
 django-formtools==2.2
 django-crispy-forms==1.11.2
 
-docker==4.4.4
+docker==5.0.0
```

Comment:
The supported versions of  python now are >=3.6 https://github.com/docker/docker-py/blob/5.0.0-release/docs/change-log.md#500

That's the breaking change, so we are good to update.